src/style.css

```diff
@@ -1,3 +1,71 @@
+
+:root {
+  --black-color: #000;
+  --blue-color: #00acdc;
+}
+
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
```

Comment:
```suggestion
  /* stylelint-disable-next-line font-family-name-quotes */
  font-family: "Roboto", sans-serif;
```

No need to disable eslint here,  just use double quotes

Reply:
![Снимок экрана 2024-03-05 180132](https://github.com/mate-academy/layout_moyo-header/assets/138056089/6bc236df-55b9-4bc1-a692-f95f56009eaa)

Я аоки не можу виправити цю помилку